pkg/docker-engine/scripts/pkg-rpm-build.sh

```diff
@@ -68,7 +68,7 @@ if [ -n "$(xx-info variant)" ]; then
 fi
 
 case "$PKG_RELEASE" in
-  centos7|centos8|centos9|oraclelinux*)
+  centos7|centos8|centos9|oraclelinux*|rockylinux8)
```

Comment:
What about rockylinux9 ?

Reply:
Hum right, excluding btrfs on rocky 8 or 9 doesn't look necessary actually. Probably since we don't install btrfs deps anymore since https://github.com/docker/packaging/pull/79. I will revisit this for centos and oraclelinux, this might have been an oversight.